inc/thorvg.h

```diff
@@ -233,19 +240,19 @@ class TVG_EXPORT Paint
      *
      * @param[in] o The opacity value in the range [0 ~ 255], where 0 is completely transparent and 255 is opaque.
      *
-     * @return @c Result::Success.
+     * @return Result::Success.
      *
-     * @note A half-translucent paint by opacity may require multiple render pass for composition, TVG recommends to avoid opacity change as possible.
+     * @note Setting the opacity with this API may require multiple renderings using a composition. It is recommended to avoid changing the opacity if possible.
```

Comment:
"render pass" is more clear.

Reply:
@mgrudzinska "render pass" "composition" are a sort of terms those are used among the graphics world.

Reply:
restored as it was originally: 
"Setting the opacity with this API may require multiple render pass for composition."